lib/main.js

```diff
@@ -115,6 +115,8 @@ const Hydrogen = {
           this.handleKernelCommand({ command: "interrupt-kernel" }),
         "hydrogen:restart-kernel": () =>
           this.handleKernelCommand({ command: "restart-kernel" }),
+        "hydrogen:refresh-all-result-bubbles": () =>
```

Comment:
> I lean towards `restart-kernel-and-re-evaluate-bubbles`. Restart is currently the only language agnostic way we have of cleanly evaluating code.

👍 I like it, It's a bit long but Atom has a great command palette so this is OK.